packages/cli/src/lib/flags/snapshotCommonFlags.ts

```diff
@@ -2,7 +2,7 @@ import {Flags} from '@oclif/core';
 import {Snapshot} from '../snapshot/snapshot';
 
 export enum PreviewLevelValue {
-  // TODO: CDX-1004: add a "None" value and remove the skipPreview flag
+  Skip = 'skip',
```

Comment:
There are also other good candidates:
* `none`
* `off`
* `ignore`

which one would make more sense?

Reply:
I'd say, `None`, when compared to the other value of the enum it's the one that make the most sense

It's not the most user-friendly way, but the most user-friendly way will I think come when we will have a `--yes` flag overall